app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  validates :title, presence: true
+  validates :content, presence: true
```

Comment:
While it is good to validate fields in the model class, do not forget to create these constraints in the schema also, currently you don't have any there. Read about Race conditions, a  quick googling would show you alot.